themes/openy_themes/openy_rose/css/styles.css

```diff
@@ -6699,6 +6699,14 @@ a[href="#step2"] {
 .global-search-results .item-list .views-field-title {
   font-weight: bold;
 }
+.global-search-results .form-item {
+  margin: 0;
+}
+
+.paragraph--type--search-api {
+  padding: 20px 15px !important;
```

Comment:
same here. Should we improve something higher to get rid of `!important` ?

Reply:
FIxed